Documentation/resources.adoc

```diff
@@ -0,0 +1,134 @@
+This document describes the following resources deployed and managed by the Cluster Monitoring Operator (CMO):
+
+* Routes
+* Services
+
+Important!
+
+In certain situations, accessing endpoints can degrade the performance and scalability of your cluster, especially if you use endpoints to retrieve, send, or query large amounts of metrics data.
+
+To avoid these issues, follow these recommendations:
+
+* Avoid querying endpoints frequently. Limit queries to a maximum of one every 30 seconds.
+* Do not try to retrieve all metrics data via the /federate endpoint. Query it only when you want to retrieve a limited, aggregated data set. For example, retrieving fewer than 1,000 samples for each request helps minimize the risk of performance degradation.
+
+## Routes
```

Comment:
```suggestion
[id="routes-cmo-services-reference"]
== Routes
```
- We need to add a unique context ID for this heading.
- Also, we should use = rather than # for all the heading levels